### DISCUSSION TITLE
Next.js app throwing an error after deploying to Digital Ocean cluster.

### DISCUSSION THREAD

**Discussion by PiotrNap** [vercel/next.js#13755] [Help]
# Bug report

I'm currently working on my app build with Kubernetes with a few services. One of them is the client side service build with next.js. When I'm on my local dev everything is working fine (see the screenshot). Unfortunately after deploying to my Digital Ocean cluster my app is throwing an error: "Module not found: Can't resolve '../components/Header' in '/app/pages'"

I'm not that experienced with next.js so I'm assuming that this may be some really silly fault on my side, but maybe not. That's why I decided to ask it here....

## Describe the bug

(as above)

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to https://github.com/PiotrNap/ticket_market
2. Click on 'download zip'
4. Run npm install
3. Install Docker, login with your profile, change username to yours in yaml files. Run "Skaffold dev"

## Expected behavior

Before I did some changes to the styling part of my app, everything was working fine. 
(I had some very basic bootrstrap styling classes applied to a few components)

## Screenshots

This is my local log from my terminal : 
![Screenshot 2020-06-04 at 13 30 36](https://user-images.githubusercontent.com/59693946/83753390-79e4bf80-a66a-11ea-93ff-82713de58679.png)

This one comes from my Digital Ocean client pod log: 
![Screenshot 2020-06-04 at 13 52 50](https://user-images.githubusercontent.com/59693946/83753501-b4e6f300-a66a-11ea-820f-5bdea231ee80.png)

## System information

- macOS 10.15.4 
- Browser Brave
- Version of Next.js: 9.4.4
- Version of Node.js: 12.16.1

**Comment by timneutkens** [comment]
Looks like you're running development mode when deploying, which is probably not what you want to do. Try using `next build` and `next start`
